flybywiresim/aircraft
# fix(efb): fuel EFB page visual glitches

Fuel EFB page bug fix

## Summary of Changes

Less visual glitches

## Screenshots (if necessary)


Not necessary

## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Check that EFB refueling animations still work when using instant board
2. Use up fuel with engines and ensure that the fuel level is decreasing in the outline


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
